### ISSUE TITLE
Add Metadata to Check

### ISSUE THREAD

**Issue by jasonmfehr** [concourse/git-resource#193]
It we be very nice to have the git metadata added to the `check` script of this resource.  If the metadata is in there, we can easily see what the next build will run without having to actually run the build or check github.

Thanks!

**Comment by vito** [comment]
This'll be done once concourse/rfcs#1 lands and we update the resource.

**Comment by jasonmfehr** [comment]
Thanks!

**Comment by taylorsilva** [comment]
This is not happening anytime soon sadly. Closing since this requires a wider discussion for changing the resource's interface, which likely won't happen.
